edx_rest_framework_extensions/tests/test_authentication.py

```diff
@@ -167,3 +186,79 @@ def test_authenticate_user_creation_with_request_exception(self):
             self.assertRaises(AuthenticationFailed, self.auth.authenticate, request)
 
         self.assertEqual(User.objects.all().count(), original_user_count)
+
+
+@override_settings(
+    ROOT_URLCONF='edx_rest_framework_extensions.tests.test_authentication',
+)
+class JwtAuthenticationTests(TestCase):
+    """ JWT Authentication class tests. """
+    def assert_jwt_status(self, issuer=None, expires=None, status_code=200):
+        """Assert that the payload has a valid issuer and has not expired."""
+        staff_user = factories.UserFactory(is_staff=True)
+        issuer = issuer or settings.JWT_AUTH['JWT_ISSUER']
```

Comment:
These settings should be documented. A link to `rest_framework_jwt` should be sufficient.